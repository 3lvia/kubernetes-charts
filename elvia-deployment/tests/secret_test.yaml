suite: test secret
templates:
  - secret.yaml
tests:
  - it: secret snapshot should match
    set:
      image.tag: latest
      name: demo-app
      namespace: core
      microserviceType: worker
      environment: dev
      replicaCount: 1
      resources:
        requests:
          cpu: 100m
          memory: 128Mi
        limits:
          cpu: 400m
          memory: 500Mi
      secret:
        dev:
          api_key: '__API_KEY_TO_BE_REPLACED_DEV__'
        test:
          api_key: '__API_KEY_TO_BE_REPLACED_TEST__'
        prod:
          api_key: '__API_KEY_TO_BE_REPLACE_PROD__'

    asserts:
      - matchSnapshot: {}
