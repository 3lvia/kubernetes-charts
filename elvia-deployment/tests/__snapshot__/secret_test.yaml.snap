secret snapshot should match:
  1: |
    apiVersion: v1
    kind: Secret
    metadata:
      name: demo-app
      namespace: core
    stringData:
      api_key: __API_KEY_TO_BE_REPLACED_DEV__
    type: Opaque
